lib/route.js

```diff
@@ -312,7 +315,9 @@ function buildRouting (options) {
       }
     }
 
-    var childLogger = logger.child({ [requestIdLogLabel]: req.id, level: context.logLevel })
+    // prevent default route
+    var serializers = context.logSerializer || {}
```

Comment:
This will allocate an unneeded object in a hot path. Would you mind adding an if to avoid adding a serializer property at all?

Reply:
@mcollina I used some functions([this commit](https://github.com/fastify/fastify/pull/1901/commits/0475f5fe82941dcbb96280c12b87eacc1c00e1e3)) to check inside object rvalue for better performed, but uses ES5 features and no longer supported by node 6. However, I changed to assign the object to a variable [this commit](https://github.com/fastify/fastify/pull/1901/commits/6638981e291d8ea42eec42980582a2fd9908d73c). 

Btw, can be good set serializers on the fly inside `pinojs`... What do you think?